src/main/java/com/hms_networks/americas/sc/extensions/json/JSONObject.java

```diff
@@ -561,7 +561,7 @@ public int length() {
     return ja.length() == 0 ? null : ja;
   }
 
-  /** Shave off trailing zeros and decimal point, if possible. */
+  /* Shave off trailing zeros and decimal point, if possible. */
```

Comment:
The method is now missing any Javadoc with this change.

Reply:
I'll bring back the docs. Changed.